pkg/oauthserver/userregistry/identitymapper/lookup.go

```diff
@@ -31,9 +31,10 @@ func (p *lookupIdentityMapper) UserFor(info authapi.UserIdentityInfo) (kuser.Inf
 	}
 
 	return &kuser.DefaultInfo{
-		Name:   u.Name,
-		UID:    string(u.UID),
-		Groups: u.Groups,
+		Name: u.Name,
+		UID:  string(u.UID),
+		// UserFor should only set name and UID, all other values are ignored
+		// Groups: u.Groups,
```

Comment:
Drop this line too ?
The comment already says it all

Reply:
Done.